Resources/Locale/en-US/job/job-names.ftl

```diff
@@ -4,6 +4,7 @@ job-name-cadet = Security Cadet
 job-name-hos = Head of Security
 job-name-detective = Detective
 job-name-scientist = Scientist
+job-name-research-assistant = research assistant
```

Comment:
```suggestion
job-name-research-assistant = Research Assistant
```
This should probably be capitalised the same as the other jobs.

Reply:
this is lowercase as this is being changed with #13769. I didn't want to create extra work with it.